File: CCC 2019/J1.py
a = 0
b = 0
a += int(input()) * 3
a += int(input()) * 2
a += int(input()) * 1
b += int(input()) * 3
b += int(input()) * 2
b += int(input()) * 1

if a > b:
  print("A")
elif b > a:
  print("B")
else:
  print("T")

File: Mock CCC 2018 Contest 5/J1.cpp
#include <bits/stdc++.h>
using namespace std;

int main() {
    char inpArray[11];
    scanf("%s",inpArray);
    string input = inpArray;
    if (input.find("CCC") != string::npos) {
        printf("NO\n");
    }
    else {
        printf("YES\n");
    }
}
